draft-ietf-quic-recovery.md

```diff
@@ -735,7 +737,7 @@ illustrate persistent congestion:
 
 The first three packets are determined to be lost when the acknowlegement of
 packet 4 is received at t=8.  The congestion period is calculated as the time
-between the oldest and newest lost packets: (3 - 0) = 3.  The duration for
+between the oldest and newest lost packets: (7 - 0) = 7.  The duration for
```

Comment:
I believe it was correct before. The time is the time between oldest and newest **lost** packets. The oldest **lost** packet was at t=3.

Reply:
Sorry, ignore me. I misread the sample packets.

Reply:
Good point, my mistake on reading the example.  It's correct as is.

Reply:
Ok, the rich diff in the browser confused me once more. I go back to my original statement. I think the original text is correct.